app/docs/0.11.x/configuration.md

```diff
@@ -201,19 +206,45 @@ http {
 
 [Back to TOC](#table-of-contents)
 
-#### Embed Kong in OpenResty
+#### Starting Kong via Nginx
 
-If you are running your own OpenResty servers, you can also easily embed Kong
-by including the Kong Nginx sub-configuration using the `include` directive
+The command `kong start` launches Kong, internally initiating an Nginx
+process. If you are running your own OpenResty servers, you may want
+to flip this process over, starting Nginx yourself and having its
+configuration launch Kong. This may also desirable in containerized
+environments, to ensure that the Nginx master process is the foreground
+process in the container, instead of the `kong` CLI command.
+
+To launch Kong from within Nginx, you need an Nginx configuration file
+which includes the Kong Nginx sub-configuration using the `include` directive
 (similar to the examples of the previous section). However, you will need the
-final configuration and not the template. For this, use the `compile` command,
-which outputs a fully-compiled Nginx sub-configuration to `stdout`:
+final configuration and not the template. To prepare this configuration file,
+use the `kong prepare` command, which generates `nginx.conf` and
+`nginx-kong.conf`:
+
+``` bash
+$ kong prepare -p /usr/local/kong -c /path/to/kong.conf 
+```
+
+The above command generates `nginx.conf` and `nginx-kong.conf` in
+`/usr/local/kong`. You can then start OpenResty with this configuration,
+possibly editing this freshly-generated `nginx.conf` file to further customize
+it.
 
 ```
-$ bin/kong compile --conf kong.conf > /usr/local/openresty/conf/nginx-kong.conf
+$ nginx -p /usr/local/kong -c /usr/local/kong/nginx.conf
+```
+
+If you already have an existing Nginx configuration for your OpenResty server,
+you can include the generated `nginx-kong.conf` to your own `nginx.conf`
+instead.
+
+Alternatively, if you created your own template with your
+[custom Nginx configuration](#custom-nginx-configuration) you can also
+use the `--nginx-conf` flag with `kong prepare`:
 
-# now start OpenResty with a configuration that "includes" nginx-kong.conf
-$ nginx -c /usr/local/openresty/conf/nginx.conf
+``` bash
```

Comment:
(Didn't run this branch myself but just double checking: this space before `bash` is accepted by the Markdown parser and will not produce some strange quirk, will it?)